light_curves/code/ztf_functions.py

```diff
@@ -97,23 +94,26 @@ def file_name(filtercode, field, ccdid, qid, basedir=None):
         assert len(files) == 1, f"found {len(files)} files. expected 1."
         return files[0]
 
-    f = f"{basedir}/field{field:06}/ztf_{field:06}_{filtercode}_c{ccdid:02}_q{qid}_{DATARELEASE}.parquet"
-    return CATALOG_ROOT + f
+    name = f"{basedir}/field{field:06}/ztf_{field:06}_{filtercode}_c{ccdid:02}_q{qid}_{datarelease}.parquet"
+    return CATALOG_ROOT + name
 
 
-def locate_objects(coords_list, labels_list, radius):
+def locate_objects(source_table, radius, datarelease=DATARELEASE, chunksize=10000):
```

Comment:
Same as above for the data release.

I also don't think the chunksize should be an argument here because changing it will very likely lead to worse performance. The time that one TAP query takes increases approx. linearly with the number of objects until you go over about 10,000 objects; beyond that it slows way down. I found this out by doing a lot of testing after trying to run it at scale (500k objects) ran into problems. There's nothing the user can do to change how the TAP query will respond, so I think it's confusing to make this an argument.

Reply:
This is being used only internally, I would instead suggest properly indicating that it's a private function, and then the new kwargs can stay (to make our life easier)